index.bs

```diff
@@ -1457,6 +1459,79 @@ Resignation from a Group</h3>
 	See “Exclusion and Resignation from the Working Group” in the W3C Patent Policy [[PATENT-POLICY]]
 	for information about obligations remaining after resignation from certain groups.
 
+<h3 id="tooling">
+Tooling</h3>
+
+	For [=W3C Groups=] operating under this Process,
+	a core operating principle is to allow access across disabilities,
+	across country borders,
+	and across time.
+	Thus in order to allow all would-be participants to effectively participate,
+	to allow future participants and observers to understand the rationale and origins of current decisions,
+	and to guarantee long-lived access to its publications,
+	W3C requires that:
+
+	<ul>
+		<li>
+			All reports, publications, or other deliverables
+			produced by the group for public consumption
+			(i.e. intended for use or reference outside its own membership)
+			<em class=rfc2119>must</em> be published and promoted at a W3C-controlled URL,
```

Comment:
in general, *must* statements benefit from being in the active form and applied to an agent - I don't know how systematically the Process Document follows that practice (although from a cursory glance, it does it often); I think it would be preferable to rephrase the MUST in this PR to say who (the group? the team? the chairs?) has what requirements .

Reply:
on the specific W3C-controlled URL piece, it would probably be useful to evaluate how this rule would apply to Web Platform Tests (where a non-insignificant amount of W3C groups effort gets invested)

Reply:
I agree in general, but it's a mix of the team contact working with the chair, editor(s), and systeam here. We could say that the [team contact | chair] must work with those people to ensure that …
?

Reply:
I agree with dsinger, the subject in an active sentence of the same meaning would be too vague to state here. Also, although this is stated in the passive voice, the actor is clearly identified because it is not a stative verb but an active one: the actor is whoever is doing the publishing or promoting. I don't think we need a change here.

(We could say at the end of the list that the W3C Team MUST help our groups adhere to these rules; but I also think it's fine without such a statement.)

Reply:
What does "and promoted" mean in this context?  (Is it unnecessary verbiage that can be cut?)

Reply:
It means that if (as likely will happen) you get a proxy from a W3C URL to github.io, then you satisfy the "published at" (there is a w3c URL); but you must also give out the W3C URL and not the github.io one when you talk about it, at least outside your group.

Reply:
"Groups must publish all content intended for public consumption at..."

The key being to know who was responsible if it went wrong, and therefore whom to talk to about fixing it.

Reply:
If a group decides to publish something elsewhere, that is an invalid decision because it falls outside the scope of what the group is authorised to do. The Staff contact is responsible for noticing that, the Chair is responsible for ensuring it doesn't happen.